packages/@ember/octane-addon-blueprint/index.js

```diff
@@ -33,4 +33,7 @@ module.exports = {
     });
 
   }
+
+  // TODO: Rename gitignore and npmignore to dot-files
```

Comment:
todo? :)

Reply:
Yep, that's one of the things that I need help with, which is why this PR is [WIP] yet.

More specifically, I'm not sure how to rename a file (to prepend a dot ".") nor which hook(s) that it should be done in. The [current/"classic" addon blueprint does this too](https://github.com/ember-cli/ember-cli/blob/v3.10.1/blueprints/addon/index.js#L162-L170) but I'm having a hard time boiling it down to just renaming a file (it does much more).